package.json

```
@@ -19,7 +19,8 @@
     "weshipit-ui:lint": "nx lint weshipit-ui",
     "weshipit-ui:test": "nx test weshipit-ui",
     "weshipit-ui:build": "nx build weshipit-ui",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "weshipit-ui:storybook": "npx nx storybook weshipit-ui"
```

Just `nx` is fine because we don't want to re-install the cli all the time
```suggestion
    "weshipit-ui:storybook": "nx storybook weshipit-ui"
```